iOSNativeLoginTemplate/iOSNativeLoginTemplate/NativeLoginView.swift

```diff
@@ -31,11 +31,33 @@ import SalesforceSDKCore
 struct NativeLoginView: View {
     @Environment(\.colorScheme) var colorScheme
     
-    @State private var username = ""
-    @State private var password = ""
+    /// The reCAPTCHA client used to obtain reCAPTCHA tokens when needed for Salesforce Headless Identity API requests
+    @EnvironmentObject var reCaptchaClientObservable: ReCaptchaClientObservable
+    
+    /// The layout type for the user's active identity flow, such as registration, forgot password or login
+    @State private var identityFlowLayoutType = IdentityFlowLayoutType.LoginViaUsernamePassword
```

Comment:
@brandonpage Without the `NavigationStack` I tried before, this enum switches between identity flow layouts here in a single view.  It really is a lot simpler and would align more to the way we can do it in the Kotlin Compose template.